src/classes/STG_PanelCustomizableRollup_CTRL.cls

```diff
@@ -56,11 +56,13 @@ public with sharing class STG_PanelCustomizableRollup_CTRL extends STG_Panel {
     }
 
     /*******************************************************************************************************
-    * @description is the org on the HH Account model
+    * @description Is the org on the HH Account model. Always returns true if HEDA + Advancement is installed
+    * @return Boolean
     */
     public Boolean isHHAccount {
         get {
-            return (STG_Panel.stgService.stgCon.npe01__Account_Processor__c == CAO_Constants.HH_ACCOUNT_PROCESSOR);
+            return (STG_Panel.stgService.stgCon.npe01__Account_Processor__c == CAO_Constants.HH_ACCOUNT_PROCESSOR)
```

Comment:
Just a small suggestion here. I think isHHAccount is used on the page to control the rendering stuff. If we put the Adv check here, it is a little bit confusing. Because it means if Adv is installed, then it is not a HHAccount model anymore. But what we actually mean is that even isHHAccount = false, if Adv is  installed, then we still want to show the customizable rollup page. So, if we want to make it more clear, then we might need to use another variable for isAdvInstalled. And check isHHAcount || isAdvInstalled.

Reply:
I totally agree, I might just rename that variable "showCRLP" or something like that, since it's only being used for that purpose right now.

Reply:
That will be great! Thanks!

Reply:
I just updated the variable name and description, keep me posted if you see anything else I should look at!